src/pages/LogDrop/FileDropper.tsx

```diff
@@ -68,19 +90,38 @@ const FileDropper: React.FC<FileDropperProps> = ({ onChangeLogType }) => {
     noKeyboard: true,
   });
 
+  let visibleUI = null;
+
+  switch (state.currentState) {
+    case "WAITING_FOR_FILE":
+      visibleUI = <FileSelector getInputProps={getInputProps} open={open} />;
+      break;
+    case "PROMPT_FOR_PARSING_METHOD":
+      visibleUI = (
+        <ParseLogSelect
+          fileName={state.fileName || ""}
+          onCancel={() => dispatch({ type: "CANCEL" })}
+          onParse={onParse}
+        />
+      );
+      break;
+    case "LOADING_FILE":
+      visibleUI = (
+        <DropzoneInnerWrapper>
+          <LoadingAnimation />
+        </DropzoneInnerWrapper>
```

Comment:
Can we delete `DropzoneInnerWrapper` by applying the style inside of `LoadingAnimation` instead?